src/Fixer/Import/FullyQualifiedStrictTypesFixer.php

```diff
@@ -63,6 +63,11 @@ final class FullyQualifiedStrictTypesFixer extends AbstractFixer implements Conf
      */
     private array $symbolsForImport = [];
 
+    /**
```

Comment:
the CI seems to be broken by some unrelated change - https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/actions/runs/7514529067/job/20457502110

Reply:
Probably related to PHPUnit `ctype` requirement in 10.5.6. It's working now.